### DISCUSSION TITLE
Add custom biome spawn controls to config

### DISCUSSION THREAD

**Discussion by PandaBearJelly** [aqariio/Fowl-Play#26] [Ideas]
A simpler way to add in custom biome settings without having to edit the .jar would be a great addition to the built-in config.

**Comment by LittleManDiscord** [comment]
I totally agree. I have mods like natures spirit and regions unexplored that add plenty of wonderful new forest biomes that are missing birds. This would be a wonderful inclusion

**Comment by aqariio** [comment]
Right now, you can make a datapack that adds biomes for the birds to spawn in! I can even make a template if you want. For these popular biome mods, I'll work on adding built-in compatibility.

**Reply by PandaBearJelly** [reply]
That's great to hear! I've been enjoying your work quite a bit and just want to see it more haha

**Reply by LittleManDiscord** [reply]
I'm very excited! and if possible, a template would be great.
